make/go.mk

```diff
@@ -27,4 +27,5 @@ TEKTON_INSTALLATION_CR_DIR=deploy/installation/tekton
 generate-assets:
 	@echo "generating assets bindata..."
 	@go install github.com/go-bindata/go-bindata/
+	@echo "generating the Tekton installation assets..."
```

Comment:
not really. I added this because I thought the process was hanging and I needed to see on which command it happened. But yes, indeed, this will probably be removed anyways.